src/OrchardCore.Modules/OrchardCore.OpenId/Views/OpenIdServerSettings.Edit.cshtml

```diff
@@ -81,7 +81,7 @@
             @T["Enable Token Endpoint"]
         </label>
     </div>
-    <span class="hint">@T["Enables action /OrchardCore.OpenId/Access/Token"]</span>
+    <span class="hint">@T["Enables action /connect/token"]</span>
```

Comment:
Assuming we want to make that configurable, I guess we should move to a parametrized call?